src/patternfly/components/Button/button.scss

```diff
@@ -74,38 +74,58 @@
   --pf-c-button--m-danger--active--Color: var(--pf-global--Color--light-100);
 
   // Link btn
+  --pf-c-button--m-link--BackgroundColor: transparent;
   --pf-c-button--m-link--Color: var(--pf-global--link--Color);
+  --pf-c-button--m-link--hover--BackgroundColor: transparent;
   --pf-c-button--m-link--hover--Color: var(--pf-global--link--Color--hover);
+  --pf-c-button--m-link--focus--BackgroundColor: transparent;
   --pf-c-button--m-link--focus--Color: var(--pf-global--link--Color--hover);
+  --pf-c-button--m-link--active--BackgroundColor: transparent;
   --pf-c-button--m-link--active--Color: var(--pf-global--link--Color--hover);
   --pf-c-button--m-link--disabled--BackgroundColor: transparent;
   --pf-c-button--m-link--m-inline--hover--TextDecoration: var(--pf-global--link--TextDecoration--hover);
   --pf-c-button--m-link--m-inline--hover--Color: var(--pf-global--link--Color--hover);
 
-
   // Plain btn
+  --pf-c-button--m-plain--BackgroundColor: transparent;
```

Comment:
do we want these as variables ? or should they just be set as `transparent`. Just seems like its adding a lot of variables when we don't want them to be overwritten

Reply:
I don't feel strongly about it, but I lean toward making the components easier to theme than less, unless it introduces a lot of overhead, which this doesn't. For example, someone could use something like a very light, semi-transparent background here in their theme.

Plus, properties like colors (text, background, border, etc) are almost always variables that can be overridden, so there is likely a developer expectation that this background could be overridden via a variable.

Reply:
Ok that sounds good to me!